src/test/run-pass/signal-alternate-stack-cleanup.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// Previously memory for alternate signal stack have been unmapped during
+// main thread exit while still being in use by signal handlers. This test
+// triggers this situation by sending signal from atexit handler.
+
+#![cfg_attr(unix, feature(libc))]
+#[cfg(unix)]
```

Comment:
You can probably remove all of these `#[cfg(unix)]` annotations by adding `// ignore-windows` to the top of the test (as it's a Unix-only test)